blackmarket/UI/Common/Textfield/TextFieldViewConfiguration.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  TextFieldViewConfiguration.swift
+//  swiftui-base
+//
+//  Created by Leandro Higa on 31/08/2022.
+//
+
+import Combine
+
+final class TextFieldViewConfiguration: ObservableObject {
+        
+    @Published var value: String = ""
+        
+    var shouldShowError: AnyPublisher<Bool, Never> {
+        Publishers.CombineLatest(isValid, Just(isEmpty))
```

Comment:
using `Just(isEmpty)` is not good here, it will emit a single value and won't change after that. So when `value` changes you will be emitting wrong signals from this publisher. You need a signal also emitting the latest value whether the value is empty or not.